requirements.txt

```diff
@@ -7,3 +7,5 @@ statsmodels==0.6.1
 scikit_learn==0.15.2
 pandas==0.17.0
 patsy==0.4.0
+ipython==4.0.0
```

Comment:
Yes, but we're rendering your iPython notebook now. This is also the reason for including nbconvert==4.0.0, which is needed to convert it to HTML.